bot/cogs/clickup.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, bot: AutoShardedBot):
         self.lists = CaseInsensitiveDict()
 
     async def on_ready(self):
-        with ClientSession() as session:
+        async with ClientSession() as session:
```

Comment:
These are out of scope. Should I remove them?

Reply:
they are probably improvements, but maybe it'd be better if gdude made the changes to this file.

Reply:
For the record, I did these so I could get aiohttp3 to work, but then later realized I should be using 2.5 with this version of discord.py rewrite.

Reply:
eh, I'll remove them.